src/main/java/io/wisetime/connector/api_client/EndpointPath.java

```diff
@@ -13,10 +13,10 @@
  */
 public enum EndpointPath {
 
-  TagDelete("/tag/:tagName/", ConnectApiRequest.HttpMethod.DELETE),
-  TagUpsert("/tag/", ConnectApiRequest.HttpMethod.POST),
-  TagAddKeyword("/tag/:tagName/keyword/", ConnectApiRequest.HttpMethod.POST),
-  TagDeleteKeyword("/tag/:tagName/keyword/:keyword", ConnectApiRequest.HttpMethod.DELETE),
+  TagDelete("/tag/delete", ConnectApiRequest.HttpMethod.POST),
+  TagUpsert("/tag", ConnectApiRequest.HttpMethod.POST),
+  TagAddKeyword("/tag/keyword", ConnectApiRequest.HttpMethod.POST),
+  TagDeleteKeyword("/tag/keyword/delete", ConnectApiRequest.HttpMethod.POST),
   TeamInfo("/team/info", ConnectApiRequest.HttpMethod.GET),
   PostedTimeSubscribe("/postedtime/subscribe", ConnectApiRequest.HttpMethod.POST);
```

Comment:
I really don't think the connector library should handle "/postedtime/subscribe". The connector requires `API_KEY` to work which means subscribing a webhook must be done first before running the connector?

Should we retain this and also add support for `unsubscribe` endpoint?

Reply:
`API_KEY` is required to subscribe a webhook. I think we can keep this in the library, and also add unsubscribe support.

Reply:
Ok, i have updated PR for unsubscribe support. thanks